spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/server/ManagementServerProperties.java

```diff
@@ -54,6 +54,12 @@ public class ManagementServerProperties {
 	 */
 	private String basePath = "";
 
+	/**
+	 * Enable management access logs to be prefixed with management_
+	 * management.server.accesslog.prefix.
+	 */
+	private boolean accesslogPrefix = true;
```

Comment:
Instead of a boolean toggle for the prefix, I'd prefer a `String` variable defaulting to `management_`. This way, people can just set it to `""` to get rid of the prefix, but also allows them to customize it.